### ISSUE TITLE
Tweaks, mods and craft suggestions for Beyond The Skies !

### ISSUE THREAD

**Issue by Migmag789** [Migmag789/Migmodpacks#3]
Hey ! In this section, you can send every kind of suggestions you have for Beyond The Skies :D

**Comment by mikolajww** [comment]
Hi, i have some UI bugs when accessing crafting tables. Nothing too big but kinda annoying [pic1](https://puu.sh/BnTQZ/e106dee664.jpg) [pic2](https://puu.sh/BnTSU/ffbf6aa957.png) [pic3](https://puu.sh/BnTSL/8bc389e490.png). Also could you tell me how to enable mod [tooltip](http://puu.sh/BnTRU/96ce9a581b.jpg) when looking at items in inventory? Tried to find it myself but I'm not really that good in modding/configs.

**Comment by Migmag789** [comment]
Thank you for the report. The mod NoMoreRecipeConflict is here because he was supposed to prevent recipes conflict. But he appears on every UI and that makes it annoying... I think I'm going to remove it later. For the trash can, I don't really know ! He wasn't here, I added a few mods, he's here. I guess it's ExtraUtils 2. I'm going to check. And for the tooltip mod, that's Mod Name Tooltip. I didn't added it because I was thinking of making a unified pack, so I was thinking that wasn't useful. If you want I'm going to add it. 
Thank you and have fun ! :D

**Comment by zerefz1998** [comment]
Is there a way to add Ender IO? The conduits are extremely useful and easy to make and using energy banks are way better than using the energy cells.

**Comment by Migmag789** [comment]
No, sorry I don't want to add EnderIO. I want to keep a tech tree in the modpack and Ender IO can't enter :(

**Comment by CrankySupertoon01** [comment]
Remove the buggy mod called Extra Planets. If you need more info search Modern Skyblock 3, Extra Planets

**Comment by CrankySupertoon01** [comment]
> The mod NoMoreRecipeConflict is here because he was supposed to prevent recipes conflict. But he appears on every UI and that makes it annoying... 

Try the mod crafting Ex. It adds a feature to do it on vanilla crafting tables.
(it doesnt support tinkers, because the spirit of the mod is a use for the vanilla one, but the mod makes it usable, so you can disable the tinkers one)

**Comment by Maxipyke** [comment]
Well but i don't see 1.7 update questions,how can i ?

**Comment by Migmag789** [comment]
The latest is 1.8

**Comment by theaxer102** [comment]
I have a problem with no sound when I enter the game World and I dont know what causes it to happen
